Src/AutoMoqUnitTest/FixtureIntegrationWithAutoConfiguredMoqCustomizationTest.cs

```diff
@@ -76,6 +76,19 @@ public void PropertiesReturnValueFromFixture()
         }
 
         [Fact]
+        public void GetOnlyPropertiesReturnValueFromFixture()
```

Comment:
Added the additional check to `VirtualMembersReturnValueFromFixture()`. Is that good enough, or is it better if I make it a different test?